api/ocm/plugin/README.md

```diff
@@ -0,0 +1,151 @@
+# The OCM Library Plugin Concept
+
+The ocm library supports a plugin mechanism to provide further variants
+for OCM and library extension points without the need of extending and recompiling
+the OCM CLI (and other applications using this library).
+
+This plugin concept is not part of the OCM specification, because it is
+a feature of this library implementation.
+
+The following extension points are supported:
+
+- Access methods
+- Uploaders
+- Downloaders
+- Actions
+- Value sets (for example for routing slip entries)
+- Config types
+- Value Merge Handler (for example for label values in delta transports)
```

Comment:
What does this mean? I am missing explanation on what these points should be able to do. Is that available somewhere else? if so we should link it here I think